float64 left_encoder
float64 right_encoder
float64 yaw_angle
float64 roll_angle
float64 pitch_angle
bool quest

